src/app/manage/ApplicationSettings.ts

```diff
@@ -216,7 +216,7 @@ export class ApplicationSettings extends Entity<number>  {
   includeGroupsInCreateEvent = new StringColumn({ includeInApi: Roles.admin });
   excludeGroupsInCreateEvent = new StringColumn({ includeInApi: Roles.admin });
 
-
+  maxDeliverisQuantityToShowForIndependent = 6;
```

Comment:
requirement changed - please set this to 4 (i.e. can see 1, 2, 3 or 4 - cannot see 5 or higher)

Reply:
might be nice to add this as a column so that the user may be able to change this setting in the app settings screen.